frontend/src/util/worksheet_utils.js

```diff
@@ -7,6 +7,10 @@ export function renderDuration(s) {
     // s: number of seconds
     // Return a human-readable string.
     // Example: 100 => "1m40s", 10000 => "2h46m"
+    if (typeof s === 'undefined'){
```

Comment:
`s` should not be a string containing 'undefined'.  The fact that this is happening means that there's a bug elsewhere.

Reply:
I double-checked the logic that generates this value. Here is the fact I collected:
1. `time` field is not filled with a real value until the bundle moves to `RUNNING` stage. This means that the `time` field will not be assigned with real value when the bundle is in `STAGED` state.
2. When frontend code tries to display the `time` field from metadata, even though `metadata.time` variable is defined, but it still hasn't been assigned with real value which leads to `typeof s === "undefined"`.
3. When frontend tries to display the `time` field using [renderDuration function](https://github.com/codalab/codalab-worksheets/blob/master/frontend/src/util/worksheet_utils.js#L6-L30), since the value of `s` is undefined, variable `y` and `d` is `NaN`. At the end the function see [Line 30](https://github.com/codalab/codalab-worksheets/blob/master/frontend/src/util/worksheet_utils.js#L30), when it concatenates `Math.round(y) + 'y' + Math.round(d) + 'd';`, since both `y` and `d` are `NaN`, the final returning result becomes `NaNyNaNd`. 

Another option to fix this could be checking `isNaN(y)` and `isNaN(d)` before the last return statement. I think checking `s` at the beginning of this function might be better as the program doesn't need to go through the rest of the logic. Let me know how you think!